dynd/__init__.py

```diff
@@ -1,155 +1,29 @@
 
-        return func
-
-    return wrap
-
-def test(verbosity=1, xunitfile=None, exit=False):
-    """
-    Runs the full DyND test suite, outputing
-    the results of the tests to  sys.stdout.
-    Parameters
-    ----------
-    verbosity : int, optional
-        Value 0 prints very little, 1 prints a little bit,
-        and 2 prints the test names while testing.
-    xunitfile : string, optional
-        If provided, writes the test results to an xunit
-        style xml file. This is useful for running the tests
-        in a CI server such as Jenkins.
-    exit : bool, optional
-        If True, the function will call sys.exit with an
-        error code after the tests are finished.
-    """
-    import os, sys, subprocess
-    import numpy
-    from .ndt.test import get_tst_module as ndt_test
-    from .nd.test import get_tst_module as nd_test
-    import unittest
-
-    print('Running unit tests for the DyND Python bindings')
-    print('Python version: %s' % sys.version)
-    print('Python prefix: %s' % sys.prefix)
-    print('DyND-Python module: %s' % os.path.dirname(__file__))
-    print('DyND-Python version: %s' % __version__)
-    print('DyND-Python git sha1: %s' % __git_sha1__)
-    print('LibDyND version: %s' % __libdynd_version__)
-    print('LibDyND git sha1: %s' % __libdynd_git_sha1__)
-    print('NumPy version: %s' % numpy.__version__)
-    sys.stdout.flush()
-    if xunitfile is None:
-        # Run all the tests
-        all_suites = []
-        for fn in os.listdir(os.path.join(os.path.dirname(__file__), 'ndt/test')):
-            if fn.startswith('test_') and fn.endswith('.py'):
-                tst = ndt_test(fn[:-3])
-                all_suites.append(unittest.defaultTestLoader.loadTestsFromModule(tst))
-        for fn in os.listdir(os.path.join(os.path.dirname(__file__), 'nd/test')):
-            if fn.startswith('test_') and fn.endswith('.py'):
-                tst = nd_test(fn[:-3])
-                all_suites.append(unittest.defaultTestLoader.loadTestsFromModule(tst))
-        runner = unittest.TextTestRunner(stream=sys.stdout, verbosity=verbosity)
-        result = runner.run(unittest.TestSuite(all_suites))
-        if exit:
-            sys.exit(not result.wasSuccessful())
-        else:
-            return result
-    else:
-        import nose
-        import os
-        argv = ['nosetests', '--verbosity=%d' % verbosity]
-        # Output an xunit file if requested
-        if xunitfile:
-            argv.extend(['--with-xunit', '--xunit-file=%s' % xunitfile])
-        # Add all 'tests' subdirectories to the options
-        rootdir = os.path.dirname(__file__)
-        for root, dirs, files in os.walk(rootdir):
-            if 'test' in dirs:
-                testsdir = os.path.join(root, 'test')
-                argv.append(testsdir)
-                print('Test dir: %s' % testsdir[len(rootdir)+1:])
-        # Ask nose to do its thing
-        return nose.main(argv=argv, exit=exit)
-
-from .config import load
```

Comment:
```load``` is first imported in```common.__init__.py```  (https://github.com/libdynd/dynd-python/pull/701/files#diff-1779beb90a5e4b146b719bfc764a56a9R5 ),  then goes back to the dynd namespace via ```from .common import *``` in ```dynd.__init__.py```.

I agree that we should move the config module either to the common module or to dynd.ndt. Perhaps as a follow-on to this PR, since there are quite some import and build changes involved.